src/renderer/shared/api/cross-chain/crossChainService.ts

```diff
@@ -0,0 +1,24 @@
+import { XCM_URL, XCM_KEY } from './common/constants';
+import { XcmConfig } from './common/types';
+
+export const fetchXcmConfig = async (): Promise<XcmConfig> => {
```

Comment:
Used js module instead of hook-like service to eliminate redundant function creation during React renders
This also makes mocking in tests much easier